analysis/topEFT/make_jsons.py

```diff
@@ -1364,6 +1383,11 @@ def main():
     #make_jsons_for_dict_of_samples(central_UL17_dict,"root://ndcms.crc.nd.edu/","2017",out_dir_central_UL,on_das=True)
     #make_jsons_for_dict_of_samples(central_UL18_dict,"root://ndcms.crc.nd.edu/","2018",out_dir_central_UL,on_das=True)
     #make_jsons_for_dict_of_samples(sync_dict,"root://ndcms.crc.nd.edu/","2017",out_dir_central_sync)
+    #Note: The following four sig samples are used to generate JSONs after being moved to ND. For now, these are the only central sig samples in hadoop.
+    #make_jsons_for_dict_of_samples(central_UL17_tZq_dict, "/hadoop","2017", out_dir_central_UL,on_das=False)
+    #make_jsons_for_dict_of_samples(central_UL18_tZq_dict, "/hadoop","2018", out_dir_central_UL,on_das=False)
+    #make_jsons_for_dict_of_samples(central_UL16_tZq_dict, "/hadoop","2016", out_dir_central_UL,on_das=False)
+    #make_jsons_for_dict_of_samples(central_UL16APV_tZq_dict, "/hadoop","2016APV", out_dir_central_UL,on_das=False)
```

Comment:
These can be removed once the dictionaries are combined.

Reply:
Hmm, I think in the past we've usually left these calls included in the code (just commented out). I agree it's not the cleanest way to handle thing, but it makes things _way_ easier to rerun in the future.

Reply:
Sounds good.